src/components/task-plan/footer.cjsx

```diff
@@ -79,32 +71,40 @@ PlanFooter = React.createClass
             >
             {'Save as Draft'}
           </AsyncButton>
-          <p>
-            An assignment in draft will not be available to students, even if the open date has passed.
-          </p>
-        </BS.Col>
 
     hasExercises = TaskPlanStore.getExercises(id)?.length
     if TaskPlanStore.isHomework(id) and not TaskPlanStore.isVisibleToStudents(id) and not hasExercises
       publishButton = null
       saveLink = null
-      selectProblems = <BS.Col sm={6} md={2}><BS.Button
+      selectProblems = <BS.Button
         bsStyle='primary'
         className='-select-problems'
         onClick={clickedSelectProblem}>Select Problems
-      </BS.Button></BS.Col>
+      </BS.Button>
 
-    <BS.Row className='footer-buttons'>
+    tips = <BS.Popover>
+      <p>
+        <strong>Publish</strong> will make the assignment visible to students on the open date.
+        If open date is today, it will be available immediately.
+      </p>
+      <p>
+        <strong>Cancel</strong> will discard all changes and return to the calendar.
+      </p>
+      <p>
+        <strong>Save as draft</strong> will add the assignment to the teacher calendar only.
+        It will not be visible to students, even if the open date has passed.
+      </p>
+    </BS.Popover>
+
+    <div className='footer-buttons'>
       {selectProblems}
       {publishButton}
-      <BS.Col sm={6} md={2}>
-        <BS.Button aria-role='close' onClick={@onCancel}>Cancel</BS.Button>
-        <p>Discard all changes and return to the calendar.</p>
-      </BS.Col>
+      <BS.Button aria-role='close' onClick={@onCancel}>Cancel</BS.Button>
       {saveLink}
-      <BS.Col sm={6} md={2}>
-        {deleteLink}
-      </BS.Col>
-    </BS.Row>
+      <BS.OverlayTrigger trigger='click' placement='top' overlay={tips} rootClose={true}>
+        <i className="footer-instructions fa fa-info-circle" onClick={@toggleTips}></i>
```

Comment:
Can this be contained in a button so teachers can tab to it (and press space to select it)?